### DISCUSSION TITLE
Meross MSS310 non configurabile in "Energia"

### DISCUSSION THREAD

**Discussion by vanzo95** [indomus/forum#1830] [HUB: Home Assistant]
Buongiorno a tutti,

Ho acquistato tre prese smart Meross MSS310. Funzionano bene e sono riuscito ad implementarle in HomeAssistant usando questa integrazione: https://github.com/albertogeniola/meross-homeassistant

L'unica cosa che non sono riuscito a fare è implementare queste tre prese nella dashboard "Energia" come Dispositivi Individuali.
Ho anche provato a creare un sensore con Integrazione secondo Riemann (in kWh), ma nulla, non vengono viste.
Qualcuno ha suggerimenti?
Grazie!

**Comment by maurvina** [comment]
![20220824_162541](https://user-images.githubusercontent.com/96523576/186444044-82b20487-aec4-470f-a0ac-5242fac59f69.jpg)
A me, funzionano, creando il sensore di Riemann.
